<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionComponentEditor</class>
 <widget class="QDialog" name="ActionComponentEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit actions</string>
  </property>
  <widget class="QTableWidget" name="tableActions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Action Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Target Entity Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delay</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Switch Index</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonAddItem">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>375</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRemoveItem">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>375</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
